bbc/simorgh
# WSTEAM1-798: Fix HCM for Media Placeholder Play Button and Consent Banner

Resolves JIRA [WSTEAM1-798](https://jira.dev.bbc.co.uk/browse/WSTEAM1-798)

Overall changes
======
Adds styling via the forced colours media query to apply styling to make the UX better for Windows HCM and Firefox colour preferences.

Code changes
======

- Adds appropriate colour changes for play button and consent banner elements with the forced-colors media query.
- Adds border to play button via the forced-colors media query.
- Updates snapshots.

Testing
======
1. Spin up the Next.js app locally
2. Visit http://localhost:7081/pidgin/live/c7p765ynk9qt
3. Check on Windows HMC and by changing Firefox colour preferences.

I have tested on both, and it looks okay to me

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
